lesson-16.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/assets/styles/stylesl-16.css">
+</head>
+<body>
+    <div class="grid-container">
+        <div class="div-header">
```

Comment:
please use meaningful clas names. No need to add "div" in the class name. 
"header" is perfect. 
same for leftbar, rightbar, main and footer